Dockerfile

```diff
@@ -8,7 +8,7 @@ MAINTAINER FearNixx Technik, <technik@fearnixx.de>
 
 RUN apt update \
 	&& apt -y upgrade \
-	&& apt install -y curl ca-certificates openssl git tar unzip bash gcc libopus-dev \
+	&& apt install -y curl ca-certificates openssl git tar unzip bash gcc libopus-dev python \
```

Comment:
I'm very sure, you meant to add [``python3``](https://packages.ubuntu.com/focal/python3) and [``python3-pip``](https://packages.ubuntu.com/focal/python3-pip)